packages/mui-base/src/TrapFocus/TrapFocus.test.js

```diff
@@ -4,6 +4,7 @@ import { expect } from 'chai';
 import { act, createRenderer, screen } from 'test/utils';
 import TrapFocus from '@mui/base/TrapFocus';
 import Portal from '@mui/base/Portal';
+import userEvent from '@testing-library/user-event';
```

Comment:
AFAIK @eps1lon has been pushing toward not using this plugin, I would be in favor of doing such as well. The logic that implements `userEvent.tab();` [looks brittle](https://github.com/testing-library/user-event/blob/d2d8a3996f3af6318153f765aedd890576ce6da3/src/utils/focus/getTabDestination.ts), I don't think that we should trust it for tests. Instead, we could continue to use the real browser tab as the existing tests are doing in:

https://github.com/mui/material-ui/blob/8d6bae2e39b9c6e28caf49e0fb930ccfba4b711f/test/e2e/index.test.ts#L118 

How about we do a follow-up to remove it?

```suggestion
```

Reply:
Didn't we already have Playwright tests for TrapFocus or any other more involved user-interaction tests? I would still recomend using that approach over using `user-event`. Especiallly for a component library. Unless you have good reason for using JSDOM + replaying events in which case I'd be curious to hear why you'd prefer JSDOm + replaying events over browser tets.

Reply:
I'd be fine with moving the meaning of this test to Playwright and removing the new dependency. If I'm to be honest, I didn't notice the e2e tests initially 😅